### PR TITLE
Add pdf rag complete example and code , Add pdf chunking complete example and code

### PR DESCRIPTION
**Pull request by rishiraj**

### CODE REVIEW

File: docs/docs/examples/chunking/README.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+### Install Indexify
+
+First, let's install Indexify using the official installation script:
+
+```bash
+curl https://getindexify.ai | sh
+```
+
+Start the Indexify server:
```
Comment (diptanu) [inline docs/docs/examples/chunking/README.md:46]: Combine this with the previous step

File: docs/docs/examples/chunking/README.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+```bash
+./indexify server -d
+```
+This starts a long-running server that exposes ingestion and retrieval APIs to applications.
+
+### Install Required Extractors
+
+Next, we'll install the necessary extractors in a new terminal:
+
+```bash
```
Comment (diptanu) [inline docs/docs/examples/chunking/README.md:56]: virtualenv

File: docs/docs/examples/chunking/README.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+extraction_graph = ExtractionGraph.from_yaml(extraction_graph_spec)
+client.create_extraction_graph(extraction_graph)
+```
+
+You can run this script to set up the pipeline:
+```bash
+python pdf_chunking_graph.py
+```
+
+## Implementing the Chunking Pipeline
```
Comment (diptanu) [inline docs/docs/examples/chunking/README.md:101]: Ingestion and Retreival from the Pipeline

File: docs/docs/examples/chunking/README.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+import requests
+from indexify import IndexifyClient
+
+def download_pdf(url, save_path):
+    response = requests.get(url)
+    with open(save_path, 'wb') as f:
+        f.write(response.content)
+    print(f"PDF downloaded and saved to {save_path}")
+
+def chunk_pdf(pdf_path):
```
Comment (diptanu) [inline docs/docs/examples/chunking/README.md:116]: retreive_chunks

**Review by diptanu — CHANGES_REQUESTED** [review]

**Comment by rishiraj** [comment]
@diptanu resolved everything

**Pull request merged**